### DISCUSSION TITLE
Logs "blocked"

### DISCUSSION THREAD

**Discussion by cpsauer** [renovatebot/renovate#10484] [Request Help]
Sorry to bug you, @rarkins, but I think I need to ask for your help.

**Most succinctly:** 
Renovate GitHub bot's logs are marked as "blocked," with no updates landing. Local runs work. Could I get your help getting things back into a working state? [Ref: Job ID 379838033]

**Story and more details:**
After months of continuous, great updates from Renovate I'd gone deep into some other parts of our codebase. But when I resurfaced, I'd noticed there hadn't been any commits from renovatebot in a while. Gah!

Usually I debug by looking at the job dashboard. So I checked the renovate job logs, but found no entries whatsoever. I'm guessing this is because no new runs had gone for a long time. 

So I manually kicked off a job from the dependency dashboard. It ran, but produced a status that was new to me: "blocked". Jobs now keep kicking off, but running into the same. And the job dashboard 404s when trying to fetch the blocked logs. 

So...I ran renovate locally and everything as smoothly as expected. 

Any idea what's wrong? We really loved having Renovates updates and would love to get it back working.

Thanks so much for all you do,
Chris

**Comment by rarkins** [comment]
@cpsauer this repo was hitting disk space limits so was moved to blocked state. I've moved it back into the queue to see if it works again.

**Reply by rarkins** [reply]
I'm seeing `done` results now in the job logs

**Reply by cpsauer** [reply]
And I'm seeing the auto-updates landing. Awesome. Great to be back unblocked.

**Reply by cpsauer** [reply]
Could I ask for a bit more EBS disk so we have some breathing room--at least until the resolution of #7840?

[There isn't an easy way to make the repo much smaller, though the only file being touched is that 20kb Bazel WORKSPACE.]

Pretty please?

**Reply by rarkins** [reply]
You're on EBS now but there's no way to allocate different sizes per repo or account

**Reply by cpsauer** [reply]
Got it. Any other ideas on ways to avoid keeping bumping into the size limits?
(I assume no chance of adding that or maybe a slightly larger EBS more more generally?)

**Reply by rarkins** [reply]
If Renovate only updates the WORKSPACE file then they implies that it never installs anything to increase the size like node_modules? Therefore any size problem is entirely due to the repo's size itself (shallow cloned too)?

**Reply by cpsauer** [reply]
Yes indeed. Definitely a big repo, even shallow. And definitely one that benefits from Renovate! Speratic failures from disk space are the one real pain point.

[Though the failures have been inconsistent, I think, even without code changes, so maybe there's some other fluctuation?]

Would really love it if there's a way to keep Renovate bot consistently happy working on the repo!

**Reply by cpsauer** [reply]
@rarkins, any last thoughts on how to better work with Renovate on a large repo, or should we consider this closed out?

**Reply by rarkins** [reply]
I think if you used LFS it would reduce the clone size because currently Renovate does not resolve them, but I'm not certain. If you have large files you could also take them out of the repo and fetch them from something like S3, or potentially even git submodules so that they can be disabled.

But more ideally we'd have a way to use git to only clone parts of repos required for cases like this, and you'd be welcome to contribute research or even code for that aspect.
